src/components/AssetDetailsPanel/stories/helper.tsx

```diff
@@ -0,0 +1,37 @@
+import { Asset } from '@cognite/sdk';
+import { action } from '@storybook/addon-actions';
```

Comment:
Remove this

Reply:
export const onAssetLoaded = (asset: Asset) => {
  console.log(asset);
}

Roman this call-back methods need Asset?

Reply:
I meant that you need to remove line 2 here